### ISSUE TITLE
Move to fsprojects?

### ISSUE THREAD

**Issue by dsyme** [caindy/ODataTypeProvider#1]
Hi @caindy ,

I just came across this type provider while dealing with fsprojects/FSharp.TypeProviders.StarterPack#121. I'll add a separate issue about that

I had been meaning to look into an updated TP for OData for some time. Would you be interested in collaborating to iterate on this further?

One thing we often to to help enable collaboration is to move projects to http://github.com/fsprojects. This helps raise the visibility of the project, helps attract contributors and also gives a "backup admin" for basic tasks. If you'd like to move the project to "fsprojects" that would be great

Let me know what you think

cc @sergey-tihon @isaacabraham

**Comment by caindy** [comment]
Hi @dsyme 

Frankly, I hadn't looked this in over a year before you opened this issue, and it was hardly anything more than an aspirational readme. I spent some time over the last couple of weekends getting back into this project. I read through the SwaggerProvider code and am starting to pattern this codebase after that.

While I'm enjoying working on this, and I feel like I am starting to get on-track with an implementation, I cannot commit to much atm. I would be happy to collaborate further and move this to fsprojects, but I wanted to emphasize just how nascent this is.

**Comment by dsyme** [comment]
@caindy OK, thanks.  If we spot someone with an active need for an updated OData provider then maybe we can join the dots here :)

**Comment by sergey-tihon** [comment]
What is the difference between this TP and http://fsprojects.github.io/FSharp.Data.TypeProviders/odata.html ?

**Comment by caindy** [comment]
Hey @sergey-tihon please checkout the Background section of the README in this repo, but long story short that TP doesn't support modern OData versions.

**Comment by albertwoo** [comment]
Is this project dead?
I am using asp.net core with OData, it is very easy to create api with a lot of query and filter automatically. 
I use SAFE Stack as the front end, so I was wandering if the Fsharp OData provider can support OData v4 then everything would be great.

**Comment by caindy** [comment]
@albertwoo Yes, though it really was never quite alive, and I haven't touched it in a couple of years. I'm leaving it in case the breadcrumbs here help someone gets interested in moving this forward. It looks like Microsoft continues to make [investments in OData](https://blogs.msdn.microsoft.com/odatateam/2018/07/03/asp-net-core-odata-now-available/), and OData is still a solid alternative for folks who haven't gone down the microservices + graphql path.
